eng/common/TestResources/setup-az-modules.yml

```diff
@@ -20,8 +20,26 @@ steps:
     displayName: (MacOS) Grant access to ~/.Azure
     condition: contains(variables['OSVmImage'], 'mac')
 
-  # New-TestResources command requires Az module
-  - pwsh: Install-Module -Name Az -Scope CurrentUser -AllowClobber -Force -Verbose
+  # This alternative invocation is being setup so that PowerShell will use
+  # TLS 1.2 when downloading from the PowerShell Gallery. We are seeing instances
+  # of failed downloads similar to reported in this article and it seemed this
+  # this helped resolve the issue. I've adding it as a conditioned step because
+  # I'm not confident that this is the solution ... so I'd like to test it on a few
+  # pipelines prior to making it the default.
+  # Article: https://techcommunity.microsoft.com/t5/windows-powershell/failed-downloading-az-and-other-modules-for-powershell/m-p/1292985
+
+
+  # New-TestResources command requires Az module with TLS 1.2
+  - pwsh: |
+      [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
There's no real downside to doing this unconditionally. It just means TLS1.2 negotiation is allowed. We had to do this in the Visual Studio setup engine because we target net45, in which case it's disabled by default. Never once caused a problem, and we had to define and use a constant with the same value as `Tls12` since net45 doesn't even have that enum.

Reply:
I'd suggest we do it unconditionally as well.